tests/integration/test_jd_adapters_branches.py

```
@@ -0,0 +1,328 @@
+@pytest.mark.ci
+def test_registry_adapt_to_jd_returns_none_when_no_adapter() -> None:
+    """adapt_to_jd returns None when no adapter can handle the item."""
+    from methodologies.johnny_decimal.adapters import AdapterRegistry
+
+    registry = AdapterRegistry()
+    item = _item("file.txt", "file.txt")
+    assert registry.adapt_to_jd(item) is None
+
+
+@pytest.mark.ci
+def test_registry_adapt_to_jd_uses_registered_adapter(default_config, fs_adapter) -> None:
```

Ruff (0.15.12)

[warning] Unused function argument: `default_config`

(ARG001)